doc/kubernetes/ray-cluster-external-redis.yaml

```
@@ -0,0 +1,136 @@
+# Ray head node service, allowing worker pods to discover the head node.
+apiVersion: v1
+kind: Service
+metadata:
+  namespace: ray
+  name: example-cluster-ray-head
+spec:
+  ports:
+  - name: client
+    protocol: TCP
+    port: 10001
+    targetPort: 10001
+  - name: dashboard
+    protocol: TCP
+    port: 8265
+    targetPort: 8265
+  - name: redis
```

As of Ray 1.11.0, the server workers contact to connect to the head is actually the "GCS Server", not Redis.
Thus, the following name is more descriptive:
```suggestion
  - name: gcs-server
```